extensions/amp-story/1.0/amp-story-draggable-drawer-header.css

```diff
@@ -74,6 +74,24 @@
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" fill="rgba(255, 255, 255, .4)" viewBox="0 0 14 14"><path d="M13.295 2.115a.997.997 0 10-1.41-1.41L7 5.59 2.115.705a.997.997 0 10-1.41 1.41L5.59 7 .705 11.885a.997.997 0 101.41 1.41L7 8.41l4.885 4.885a.997.997 0 101.41-1.41L8.41 7l4.885-4.885z"/></svg>') !important;
 }
 
+.i-amphtml-story-page-attachment-domain-label {
+  background: #fff !important;
+  color: rgba(0, 0, 0, 0.87) !important;
+  display: block !important;
+  font-family: 'Roboto', sans-serif !important;
+  font-size: 12px !important;
+  overflow: hidden !important;
+  padding: 10px 20px !important;
+  text-align: center !important;
+  text-overflow: ellipsis !important;
+  white-space: nowrap !important;
+}
+
+[theme="dark"].i-amphtml-story-page-attachment-domain-label {
+  background: #202125 !important;
+  color: #fff !important;
```

Comment:
here and above, you should be able to use `background: var(--i-amphtml-draggable-drawer-background-color) !important` so you don't need to repeat the color / override for the dark theme